.github/actions/linux-test/action.yml

```diff
@@ -29,15 +29,20 @@ runs:
         tar -xzf build.tar.gz
     - name: Run unit tests
       shell: bash
+      # always run tests as cmake target coverage
+      # does not return error if a test fail
       run: |
+        cd cpp/build/tests
+        sudo chmod a+x memilio-test
+        ./memilio-test  --gtest_output="xml:report.xml"      
+    - name: Compute code coverage
+      shell: bash
+      # compute code coverage
+      run: |    
```

Comment:
I think the logic in the lines afterwards starting with  if-no-files-found: needs to be changed accordingly.